### ISSUE TITLE
The Loss_depth/depth_encoder is 0 when i train distillation policy

### ISSUE THREAD

**Issue by LargeNumberZZ** [chengxuxin/extreme-parkour#26]
![Screenshot from 2024-01-28 11-26-20](https://github.com/chengxuxin/extreme-parkour/assets/135707287/893c3c07-0cdc-4fec-9031-397b89937818)
Hi! Thanks for you great job! And I want to know that is it normal for this loss to be 0 all the time？

**Comment by BaiHiAN** [comment]
> ![Screenshot from 2024-01-28 11-26-20](https://github.com/chengxuxin/extreme-parkour/assets/135707287/893c3c07-0cdc-4fec-9031-397b89937818) Hi! Thanks for you great job! And I want to know that is it normal for this loss to be 0 all the time？

Hello, may I ask approximately how much the depth_actor_loss you trained will converge to in the end?

**Comment by sky-98k** [comment]
I also noticed this problem. In the file on_policy_runner.py, the author sets depth_encoder_loss = 0. I don't understand why, but obviously, this is not correct.
